python/sglang/srt/sampling/sampling_batch_info.py

```diff
@@ -98,7 +98,14 @@ def from_schedule_batch(cls, batch: ScheduleBatch, vocab_size: int):
         )
         sampling_seed = (
             torch.tensor(
-                [r.sampling_params.sampling_seed for r in reqs],
+                [
```

Comment:
I think we shouldn't change it here. We should put the default value in the init function of `sampling_params`

Reply:
i wonder if it really works because we have/the user might supply the "sampling_seed" field in JSON, so when dataclass deserializes it, it will put `None` here anyways? (not entirely sure, will try)

Reply:
```
        if self.preferred_sampling_params:
            sampling_kwargs = {**self.preferred_sampling_params, **obj.sampling_params}
        else:
            sampling_kwargs = obj.sampling_params
        sampling_params = SamplingParams(**sampling_kwargs)
```

yeah there's a kwargs unpack.. that's why `None` gets passed into the init function in the first place